packages/core/rollup.utils.js

```diff
@@ -96,12 +98,12 @@ export const globalStyles = config => {
         file: resolve(output).replace(resolve(config.baseDir), resolve(config.outDir)).replace(extname(output), '.css'),
       },
       plugins: [
-        styles({ mode: 'extract', plugins: [autoprefixer] }),
+        styles({ mode: 'extract', plugins: [autoprefixer({ flexbox: false })] }),
```

Comment:
Just checking. We are turning off autoprefixing for flexbox styles because all our supported browsers support it without prefixing?

Reply:
Yeah, it was generating old edge (pre-chromium) -ms-* prefixes. I initially removed the autoprefixer plugin but that made it worse since the sass styles plugin uses autoprefixer by default with a even older browser set which generated even more old browser prefixes. So the end solution was to keep the plugin but disable the flexbox prefixing.